driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/fchmod.bpf.c

```diff
@@ -6,7 +6,6 @@
  */
 
 #include <helpers/interfaces/fixed_size_event.h>
-#include <helpers/interfaces/variable_size_event.h>
```

Comment:
Minor: seems to be a fixup and maybe fold back into the [new(modern_bpf): add fchmod syscall ](https://github.com/falcosecurity/libs/pull/509/commits/bfb7f12152cc20d55b0d076066700a8993be8f07) commit.